thirdparty/harfbuzz/NEWS

```diff
@@ -1,3 +1,117 @@
+Overview of changes leading to 3.0.0
```

Comment:
We could likely remove the `NEWS` file, usually we include only the bare minimum for the engine's needs. Users interested in what's new in the latest harfbuzz version we vendor can refer to the harfbuzz repo.

Reply:
Removed